podcasts/Onboarding/Plus/PlusLandingViewModel.swift

```diff
@@ -136,6 +137,6 @@ extension PlusLandingViewModel {
     @ViewBuilder
     private static func view(with viewModel: PlusLandingViewModel) -> some View {
         UpgradeLandingView(viewModel: viewModel)
-            .setupDefaultEnvironment()
+            .setupDefaultEnvironment(theme: Theme.init(previewTheme: .light))
```

Comment:
Is this change intentional? Or it was committed by mistake?

Reply:
Yes it was intentional, to show the UpgradeLandingView, that is used on onboarding and has a coloured background   always using the light theme.

![simulator_screenshot_6EBA8A9F-CFD9-4EC3-A163-6435E80C39E4](https://github.com/Automattic/pocket-casts-ios/assets/651601/b042f3a7-3357-4d81-b919-7f47d9e82aab)